kube_apiserver_metrics/README.md

```diff
@@ -21,7 +21,7 @@ Annotations:       ad.datadoghq.com/endpoint.check_names: ["kube_apiserver_metri
                    ad.datadoghq.com/endpoint.instances:
```

Comment:
```suggestion
                   ad.datadoghq.com/endpoints.instances:
```
could be out of the scope of this PR but this is important to fix

Reply:
same for `check_names` and `init_configs`

Reply:
cc @Simwar

Reply:
let's open a second PR for any necessary documentation update unrelated to that fix